python/py-bpython/Portfile

```diff
@@ -25,7 +25,12 @@ checksums               rmd160  6a57d45732b0f756668a481c99e8c3135c741e67 \
 python.versions         27 37 38 39 310
 
 if {${name} ne ${subport}} {
-    livecheck.type          none
+    post-patch {
+        # fix version detection
+        set versionpath ${worksrcpath}/bpython/_version.py
+        copy ${filespath}/_version.py ${versionpath}
+        reinplace "s/__VERSION__/${version}/" ${versionpath}
+    }
```

Comment:
This doesn't look related to the changes described in the PR.

Reply:
@lpsinger without this changes this port simple doesn't build anymore.

Reply:
If you would like I may keep it as dedicated commit.

Reply:
BTW I've kept it as dedicated commit but haven't mentored it inside PR description. This was changed.

Reply:
I don't have enough information here to review this change. How did the build fail? How does this fix it?